templates/page.html

```diff
@@ -185,19 +185,20 @@ <h3>Select a REANA cluster...</h3>
         <h3>...or install your own</h3>
         <figure class="figure">
           <div class="command-line">
+            <!-- FIXME: Should we recommend kubectl 1.21 already? -->
             <div class="line comment"># install kubectl 1.16+, kind 0.8+ and helm 3.0+</div>
             <div class="line"><span class="prompt">$</span>sudo dpkg -i kubectl*.deb kind*.deb kubernetes-helm*.deb</div>
             <div class="line comment"># create Kubernetes cluster (or use your own!)</div>
-            <div class="line"><span class="prompt">$</span>wget https://raw.githubusercontent.com/reanahub/reana/maint-0.7/etc/kind-localhost-30443.yaml</div>
+            <div class="line"><span class="prompt">$</span>wget https://raw.githubusercontent.com/reanahub/reana/0.8.0/etc/kind-localhost-30443.yaml</div>
```

Comment:
Let's use `maint-0.8` branch name instead of `0.8.0` tag name for reasons mentioned in https://github.com/reanahub/docs.reana.io/pull/97/files/195711b53c53fc1c8ea5e3ddf3c0d27bbe82d789..df0d797f431cadc7a9e99a0f8b8e67ba21309545#r759177949